pkg/asset/kubeconfig/stock.go

```diff
@@ -0,0 +1,46 @@
+package kubeconfig
+
+import (
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/tls"
+)
+
+// Stock is the stock of the kubeconfig assets that can be generated.
+type Stock interface {
+	Kubeconfig() asset.Asset
```

Comment:
I'd rather have `KubeconfigAdmin()` here (same reasons as [this comment][1]).  This namespacing probably applies to other names too.

[1]: https://github.com/openshift/installer/pull/160/commits/17a26e77bb626340b3b0e46b3d498f3f1f3f3621#r213863667

Reply:
done